cmd/state-custom-installer/runUnInstall.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+
+	"github.com/ActiveState/cli/internal/osutils"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/subshell"
+	"github.com/ActiveState/cli/internal/subshell/sscommon"
+)
+
+// NOTE: From: internal/runners/clean/uninstall.go
+
+func runUnInstall(out output.Outputer, params *Params) error {
+	// This is the deploy id constant: DeployID RcIdentification = RcIdentification{
+	installToDir := params.path
+	// Should look for license file to see if it exists before we delete the install directory
+	uninstallFileCheck := filepath.Join(installToDir, "LICENSE.txt")
+
+	if !fileutils.FileExists(uninstallFileCheck) {
+
+		confirmUninstall, err := getConfirmation("Directory does not look like an install directory, are you sure you want to proceed?")
+		if err != nil {
+			return err
+		}
+
+		if !confirmUninstall {
+			return errs.New("ActiveState license not found in uninstall directory. Please specify a valid uninstall directory.")
+		}
+
+	}
+
+	cfg, err := config.New()
+	if err != nil {
+		return err
+	}
+
+	err = removeEnvPaths(cfg)
```

Comment:
It looks like uninstall is ONLY removing env config but otherwise leaves the install dir itself intact. Is that intended?